nixos/modules/virtualisation/containers.nix

```diff
@@ -318,6 +319,16 @@ let
       '';
     };
 
+    hostPort = mkOption {
+      type = types.nullOr types.string;
```

Comment:
First of all, `types.string` is deprecated. And second, this seems to be accepting only an integer, am I correct? If yes, why not use `types.int` instead?